# generated by patch-package 6.4.14
#
# declared package:
#   @1inch/cross-chain-sdk: 0.1.15
#
diff --git a/node_modules/@1inch/cross-chain-sdk/dist/cjs/chains.js b/node_modules/@1inch/cross-chain-sdk/dist/cjs/chains.js
index 4b4e3a6..8f05a5d 100644
--- a/node_modules/@1inch/cross-chain-sdk/dist/cjs/chains.js
+++ b/node_modules/@1inch/cross-chain-sdk/dist/cjs/chains.js
@@ -14,7 +14,9 @@ exports.SupportedChains = [
     fusion_sdk_1.NetworkEnum.ZKSYNC,
     fusion_sdk_1.NetworkEnum.LINEA,
     fusion_sdk_1.NetworkEnum.SONIC,
-    fusion_sdk_1.NetworkEnum.UNICHAIN
+    fusion_sdk_1.NetworkEnum.UNICHAIN,
+    fusion_sdk_1.NetworkEnum.NEAR,
+    fusion_sdk_1.NetworkEnum.NEAR_TESTNET
 ];
 const isSupportedChain = (chain) => exports.SupportedChains.includes(chain);
 exports.isSupportedChain = isSupportedChain;
diff --git a/node_modules/@1inch/cross-chain-sdk/dist/esm/chains.js b/node_modules/@1inch/cross-chain-sdk/dist/esm/chains.js
index 8f80835..1897c9d 100644
--- a/node_modules/@1inch/cross-chain-sdk/dist/esm/chains.js
+++ b/node_modules/@1inch/cross-chain-sdk/dist/esm/chains.js
@@ -11,7 +11,9 @@ export const SupportedChains = [
     NetworkEnum.ZKSYNC,
     NetworkEnum.LINEA,
     NetworkEnum.SONIC,
-    NetworkEnum.UNICHAIN
+    NetworkEnum.UNICHAIN,
+    NetworkEnum.NEAR,
+    NetworkEnum.NEAR_TESTNET
 ];
 export const isSupportedChain = (chain) => SupportedChains.includes(chain);
 //# sourceMappingURL=chains.js.map
\ No newline at end of file
diff --git a/node_modules/@1inch/cross-chain-sdk/dist/types/src/chains.d.ts b/node_modules/@1inch/cross-chain-sdk/dist/types/src/chains.d.ts
index 8fcab96..eb34dac 100644
--- a/node_modules/@1inch/cross-chain-sdk/dist/types/src/chains.d.ts
+++ b/node_modules/@1inch/cross-chain-sdk/dist/types/src/chains.d.ts
@@ -1,6 +1,6 @@
 import { NetworkEnum } from '@1inch/fusion-sdk';
 import { TupleToUnion } from './type-utils';
-export declare const SupportedChains: readonly [NetworkEnum.ETHEREUM, NetworkEnum.POLYGON, NetworkEnum.BINANCE, NetworkEnum.OPTIMISM, NetworkEnum.ARBITRUM, NetworkEnum.AVALANCHE, NetworkEnum.GNOSIS, NetworkEnum.COINBASE, NetworkEnum.ZKSYNC, NetworkEnum.LINEA, NetworkEnum.SONIC, NetworkEnum.UNICHAIN];
+export declare const SupportedChains: readonly [NetworkEnum.NEAR, NetworkEnum.NEAR_TESTNET, NetworkEnum.ETHEREUM, NetworkEnum.POLYGON, NetworkEnum.BINANCE, NetworkEnum.OPTIMISM, NetworkEnum.ARBITRUM, NetworkEnum.AVALANCHE, NetworkEnum.GNOSIS, NetworkEnum.COINBASE, NetworkEnum.ZKSYNC, NetworkEnum.LINEA, NetworkEnum.SONIC, NetworkEnum.UNICHAIN];
 type UnsupportedChain = Exclude<NetworkEnum, TupleToUnion<typeof SupportedChains>>;
 export type SupportedChain = Exclude<NetworkEnum, UnsupportedChain>;
 export declare const isSupportedChain: (chain: unknown) => chain is SupportedChain;
